modules/react/select/lib/SelectCard.tsx

```diff
@@ -1,20 +1,21 @@
 import {Combobox} from '@workday/canvas-kit-react/combobox';
-import {createStyles, px2rem} from '@workday/canvas-kit-styling';
-import {mergeStyles} from '@workday/canvas-kit-react/layout';
+import {createStencil, handleCsProp, px2rem} from '@workday/canvas-kit-styling';
 import {useSelectCard} from './hooks/useSelectCard';
 import {useSelectModel} from './hooks/useSelectModel';
 import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
 
-const selectCardStyles = createStyles({
-  maxHeight: px2rem(300),
+export const selectCardStencil = createStencil({
+  base: {
+    maxHeight: px2rem(300),
```

Comment:
It seems like `maxHeight` should be a Stencil variable in `MenuCard` since menu cards often need some type of overflow height. We don't want them to be too large. There should be some default. I don't want the CSS variable to be passed via JavaScript because that would restrict overrides via CSS.

It would looks something like:

```js
base: {
  [menuCardStencil.vars.maxHeight]: px2rem(300),
]
```

I don't know if `menuCard` should have a default maxHeight... It seems like it should. Menus aren't always popups, but a `MenuCard` is always part of a popup.

Reply:
give it another look when you get a second